ajax/libs/tweenjs/package.json

```diff
@@ -2,21 +2,32 @@
   "name": "tweenjs",
   "filename": "tweenjs.min.js",
   "version": "0.6.2",
-  "description": "TweenJS is a Javascript library for tweening and animating HTML5 and Javascript properties.",
-  "homepage": "http://www.createjs.com/#!/TweenJS",
+  "description": "TweenJS is a simple tweening library for use in Javascript. It was developed to integrate well with the EaselJS library, but is not dependent on or specific to it (though it uses the same Ticker and Event classes by default). It supports tweening of both numeric object properties & CSS style properties.",
   "keywords": [
-    "canvas",
+    "createjs",
+    "tweenjs",
+    "gskinner",
+    "javascript",
     "html5",
-    "animation"
+    "tween",
+    "motion",
+    "ease",
+    "animate"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/CreateJS/TweenJS.git"
+    "url": "git+https://github.com/CreateJS/TweenJS.git"
   },
   "license": "MIT",
-  "authors": [
-    "gskinner",
-    "lannymcnie",
-    "wdamien"
+  "author": "gskinner.com, inc.",
+  "homepage": "https://createjs.com/tweenjs/",
+  "npmName": "tweenjs",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```

Comment:
Need to modify this to `**/!(README.md)` ( or `**/!(*.md)` if you could put any other markdown file under `dist` in the future) to exclude `README.md`

Reply:
@wdamien Please help modify the pattern to `**/!(*.md)`
Thank you.
(Except for this part, I think this PR is GOOD!)